assets/js/modules/adsense/datastore/ad-blocking-recovery.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * `modules/adsense` data store: ad-blocking-recovery.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import invariant from 'invariant';
+
+/**
+ * Internal dependencies
+ */
+import API from 'googlesitekit-api';
+import { createFetchStore } from '../../../googlesitekit/data/create-fetch-store';
+
+const fetchAdBlockingRecoveryTagStore = createFetchStore( {
+	baseName: 'adBlockingRecoveryTag',
+	controlCallback: ( { accountID } ) => {
+		return API.get(
+			'modules',
+			'adsense',
+			'ad-blocking-recovery-tag',
+			{ accountID },
```

Comment:
This datapoint should always return the tag for the connected account – there wouldn't be a case where we'd need to request these tags for another account; nor should we. This is safe to do similar to other datapoints used by setup/settings since the response won't be cached.